kernel/patches/virtio-fs-v0.3.x/0001-net-virtio_vsock-Fix-race-condition-between-bind-and.patch

```diff
@@ -0,0 +1,49 @@
+From 6f486b9250067b788df6dad1e001a8301305a46c Mon Sep 17 00:00:00 2001
```

Comment:
is `virtio-fs-v0.3.x` a new directory? does `build-kernel` look for patches in this directory automatically?

Reply:
That's what @jcvenegas told me. Apparently, when Kata builds experimental kernel, it would look for a directory with the same name as the version.
Should it be `virtio-fs-v0.3.x` or `virtio-fs-v0.3`?

Reply:
Yes, it will find for that dirctory based in the kernel version defined in versions.yaml 
https://github.com/kata-containers/runtime/blob/master/versions.yaml#L163 seems that is not the more elegant tag.

Reply:
oks, lgtm, thanks